fem/qinterp/det.cpp

```diff
@@ -133,12 +128,24 @@ static void Det3D(const int NE,
    double *GM = nullptr;
    if (!SMEM)
    {
-      d_buff->SetSize(2*MSZ*GRID);
+      const DeviceDofQuadLimits &limits = DeviceDofQuadLimits::Get();
+      const int max_q1d = T_Q1D ? T_Q1D : (SMEM ? limits.MAX_DET_1D : limits.MAX_D1D);
+      const int max_d1d = T_D1D ? T_D1D : (SMEM ? limits.MAX_DET_1D : limits.MAX_Q1D);
```

Comment:
`SMEM` is `false` here, so the expressions can be simplified a bit.

Reply:
Fixed in [b89295d](https://github.com/mfem/mfem/pull/3722/commits/b89295dec791764fc5d4890c1cac8692b2025fe0)